structure-of-work.md

```diff
@@ -12,5 +12,20 @@ You acknowledge and agree that:
 
 
 ## Definition of "_Done_"
```

Comment:
This is referenced from the freelance agreements and my initial thought was to have some more information on structure our work and how we work together within the same file. 
Following this week it's pretty apparent the definition of done needs to be split out into its own file.